integration/terraform/ec2/linux/main.tf

```diff
@@ -36,7 +36,7 @@ resource "aws_instance" "integration-test" {
 
 data "aws_ami" "latest" {
   most_recent = true
-  owners      = ["self"]
+  owners      = ["self", "506463145083"]
```

Comment:
Where's this new owner string coming from?

Reply:
This is the cw agent open source test account. You need this to be able to find the ami I made public from the image builder pipeline.